theopensystemslab/planx-new
# fix: ensure unauthorised public users can select from `blpu_codes` table to populate PropertyInformation

This one came up in #help-issues: when testing Doncaster FOI _not signed into the editor_, Jenna was unexpectedly being asked property type questions manually. When testing signed in, the description from the BLPU codes lookup was correctly populating automatically. 

The public role can select from `blpu_codes`, so this ensures `useQuery()` now uses the `publicClient` rather than default authorised client.

To test:
- Search 45 Thealby Gardens, DN4 7EQ [here](https://2229.planx.pizza/doncaster/find-out-if-you-need-planning-permission/preview) - confirm "Property type" populates with "Detached" automatically
